build.sbt

```diff
@@ -71,7 +71,8 @@ lazy val `core-client` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
   .settings(
     name := s"$baseName-core-client",
     libraryDependencies ++= Seq(
-      "org.typelevel" %%% "otel4s-semconv-experimental" % otel4sV
+      "org.typelevel" %%% "otel4s-semconv-experimental" % otel4sV,
+      "org.http4s" %%% "http4s-dsl" % http4sV % Test,
```

Comment:
I was about to complain that this only needs to be a test dependency after getting to the bottom, and scrolled back up, and it already was. :tada:

Reply:
I really need to fix the scalafmt conf to allow trailing commas on single lines